.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you can remove/untrack or can use .gitignore as no need of this file in remote repo